File: Assignment__19/Assignment_2.c
#include<stdio.h>
#include<stdlib.h>

int Frequency(int Arr[] , int iLength)
{
    int iCnt = 0 , iCount1 = 0,iCount2 = 0, Frequency = 0 ;
    for(iCnt = 0 ; iCnt < iLength;iCnt++)
    {
        if(Arr[iCnt] % 2 == 0)
        {
            iCount1 = iCount1 + 1;
        }
        else 
        {
            iCount2 = iCount2 + 1;
        }
    }
    Frequency = iCount1 - iCount2;
    printf("(%d - %d)\n",iCount1,iCount2);
    return Frequency;

}
int main()
{
    int iSize = 0 , iCnt = 0 ;
    int iRet = 0 ;
    int *ptr = NULL;

    printf("Enter number of Elements :  ");
    scanf("%d",&iSize);

    ptr = (int *)malloc(iSize * sizeof(int));
    if(ptr == NULL)
    {
        printf("Unable to locate memory : \n");
        return -1;
    }

    printf("Enter %d numbers : \n",iSize);
    for(iCnt = 0  ; iCnt < iSize ; iCnt++)
    {
        printf("Enter %d numbers : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }

    iRet = Frequency(ptr , iSize); 
    printf("Result is : %d \n",iRet);

    free(ptr);

    return 0 ;
}

File: Assignment__19/Assignment_java/Assignment_2.java

import java.util.*;

class Difference
{
    public int Frequency(int []Arr , int iLength)
    {
        int iCountEven = 0 , iCountOdd = 0, iFreq = 0;

        for(int iCnt = 0 ; iCnt < iLength ; iCnt++)
        {
            if(Arr[iCnt] % 2 == 0 )
            {
                iCountEven = iCountEven + 1 ;
            }
            else 
            {
                iCountOdd = iCountOdd + 1 ;
            }
        }
        System.out.println();
        System.out.println("("+iCountEven+" - "+iCountOdd+")");
        iFreq = iCountEven - iCountOdd;
        return iFreq;
    }
}
class Assignment_2
{

    public static void main(String A [])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iSize = sobj.nextInt();
    

        int []ptr = new int[iSize];

        System.out.println("Enter "+iSize+" Elements : ");

        for(int iCnt = 0 ; iCnt < iSize; iCnt++)
        {
            System.out.print("Enter elements "+ (iCnt + 1) +" : ");
            ptr[iCnt] = sobj.nextInt();
        }
        Difference dobj = new Difference();
        int iRet = dobj.Frequency(ptr , iSize);
        System.out.println("Result is : "+iRet);
    }
}